students/douglas_klos/session2/series.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+# Douglas Klos
+# January 20th, 2019
+# Python 210, Assigment 2
+# series.py
+
+
+def fibonacci(n):
+    """
+    recursively computes the nth Fibonacci number
+    fib(n) = fib(n-2) + fib(n-1)
+    """
+    if (n <= 1):
+        return n
+    return fibonacci(n-2) + fibonacci(n-1)
+
+
+def lucas(n):
+    """
+    recursively computes the nth Lucas number
+    lucas(n) = lucas(n-2) + lucas(1).
+    when n == 0, lucas = 2 and when n == 1, lucas = 1
+    """
+    if (n == 0):
+        return 2
+    if (n == 1):
+        return 1
+    return lucas(n-2) + lucas(n-1)
+
+
+def sum_series(n, n0=0, n1=1):
+    """
+    recursively computes the nth value of a summation series.
+
+    :param n0=0: value of zeroth element in the series
+    :param n1=1: value of first element in the series
+    This way, if no arguments are passed, it defaults to a Fibonacci sequence.
+
+    This function generalizes the fibonacci() and the lucas(),
+    so that this function works for any first two numbers for a sum series.
+    """
+
+    if (n == 0):
+        return n0
```

Comment:
Its hard here with mathematics notation making sense here, but you'll want to avoid non descriptive variable names in the future.  Or having a several variables where the only thing that differs is a number at the end, like `str1` and `str2`.  That doesn't tell me what that string is.